### DISCUSSION TITLE
Graph QL rate limit exceeded

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#23872] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Not hitting the following error:

```
DEBUG: Graph QL rate limit exceeded. (repository=platform-engineering-org/.github)
       "res": {
         "errors": [
           {
             "type": "RATE_LIMITED",
             "message": "API rate limit exceeded for user ID 137812415."
           }
         ]
       }
```

### What you tried so far.

Waiting to the runner run which doesn't hit this error

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by jamietanna** [comment]
Closing due to age, and the fact that it's an issue with the user account being used, rather than Renovate
